tests/cases/fourslash/renameReExportDefault.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /a.ts
+////export { default } from "./b";
+////export { default as b } from "./b";
+////import { default as b } from "./b";
+////import [|b|] from "./b";
+
+// @Filename: /b.ts
+////const [|b|] = 0;
+////export default [|b|];
+
+const [r0, r1, r2] = test.ranges();
+verify.renameLocations(r0, [r0]);
+verify.renameLocations([r1, r2], [r0, r1, r2]);
```

Comment:
seems like r0 shouldnt be part of this. It is not ```import { b} from``` syntax which would need rename.

Reply:
It doesn't strictly need a rename, but it seems appropriate that if you `export default foo` and `import foo`, you intend for these two to continue having the same name? `export default bar` and `import foo` will work at runtime, but it makes the code confusing if the symbol now has multiple names.

Reply:
so why not the rename the `b` in `export { default as b } from "./b"` as well?

Reply:
Good idea. :+1: